troposphere/ec2.py

```diff
@@ -11,9 +11,9 @@
 )
 
 try:
-    from awacs.aws import Policy
+    from awacs.aws import PolicyDocument
```

Comment:
I'd been holding off on this PR in order to verify backward compatibility. I think it would be best to change the troposphere/*.py files to use both Policy and PolicyDocument.
```
try:
    from awacs.aws import Policy, PolicyDocument
    policytypes = (dict, Policy, PolicyDocument)
except ImportError:
    policytypes = dict,
```
This allows existing code to still function while supporting PolicyDocument and promoting its use in the examples.

Reply:
Yeah, you're right.  Since PolicyDocument is a subclass of Policy, Policy would no longer be valid.

But forward compatibility should also be considered.  Let's assume that awacs 1.0 drops Policy.  The current release of troposphere would not be compatible with it.  Therefore I propose the following code:

```python
try:
    from awacs.aws import Policy, PolicyDocument
    policytypes = (dict, Policy, PolicyDocument)
except ImportError:
    try:
        from awacs.aws import PolicyDocument
        policytypes = (dict, PolicyDocument)
    except ImportError:
        policytypes = dict,
```

That are imho too many lines of code that would be duplicated over many files. I therefore also propose to introduce a new module `troposphere.compat` (like eg. [django rest framework](https://github.com/encode/django-rest-framework/blob/master/rest_framework/compat.py)) and import `policytypes` from there.

Reply:
Yes, adding it via a compat library would be good. Thanks!